### ISSUE TITLE
Behavior Score 

### ISSUE THREAD

**Issue by Anime0555** [jeffhill/Dota2#638]
### Description

Low behavior score players like me are getting 45min-1hour+ queueing time per matches (im 5k SEA)
and its too hard to increase behavior score, i have below 2k. My last 2 conduct summary was 20 commends average, 0 abandon and <3 reports and i get only 300 or 400 scores added to my behavior score wtf

Its exponential, its bad... how am i supposed to increase the behavior score if my queue time is 1 hour? 
btw 9/10 ready - 1 decline - happens like 5 times in a row, good luck finding a game in 5hours

**Comment by Uwila** [comment]
I suspect it's not a bug and just part of the punishment honestly. You may not like it, but you weren't supposed to like it. You were supposed to not let it get this far, and now it's somewhat of a you-problem.

**Comment by geras256** [comment]
If you have 2k Behavior Score, that means you are extremely toxic and are griefing a lot. Thus, you are now punished, and I think you might even get perma banned. I feel no remorse.

**Comment by SnabrielSnarsch** [comment]
intended behavior

**Comment by real-vaLor** [comment]
Not even joking, you can easily get behavior score by simply muting all and not saying anything. Requires no additional effort, simply play. In a few weeks time you'd probably hit a decent level

**Comment by OMEGALULW** [comment]
> If you have 2k Behavior Score, that means you are extremely toxic and are griefing a lot. Thus, you are now punished, and I think you might even get perma banned. I feel no remorse.

No, it means griefing idiots who ruin your games are reporting you as part of the griefing
